docs/integration/kafka.md

```diff
@@ -1,8 +1,191 @@
 title: Kafka Monitoring Integration
 description: Monitor Kafka metrics for brokers, producers, and consumers, consumer lag and offset monitoring by consumer group, topic, or partition, and more. Our cloud and on-premises tools provide out of box Kafka graphs, reports and custom dashboards with built-in anomaly detection, threshold, and heartbeat alerts as well as easy chatops integrations
 
+
+Sematext offers simple Kafka monitoring agent written in Java and Golang with minimal CPU and memory overhead. It's easy to install and require no changes in the Kafka source code or your application's source code.
```

Comment:
It's easy to install and **requires** * no changes in the Kafka source code or your application's source code.

Reply:
Actually @otisg mentioned that it should be *required*.

Reply:
Here's another variation:

> Sematext has a simple Kafka monitoring Agent written in Java and Go with minimal CPU and memory overhead. It's easy to install and doesn't require any changes to the Kafka source code or your application's source code.

Reply:
Oh, if I said "required", I made a typo.  I meant requires :)
+1 for Adnan's variant, even cleaner.

Reply:
Adjusted.